### PR TITLE
Remove X-Frame-Options header for kuviz visualizations

### PR DESCRIPTION
**Pull request by Jesus89**
Fixes https://github.com/CartoDB/cartodb/issues/15019, https://github.com/CartoDB/cartoframes/issues/1114.

This allows to embed a kuviz visualization inside an iframe.

### CODE REVIEW

File: app/controllers/carto/kuviz/visualizations_controller.rb [hunk 1]
```diff
@@ -1,10 +1,14 @@
 module Carto
   module Kuviz
     class VisualizationsController < ApplicationController
+      include Carto::FrameOptionsHelper
 
       ssl_required
 
+      before_filter :x_frame_options_allow, only: [:show, :show_protected]
```
Comment (alrocar) [inline app/controllers/carto/kuviz/visualizations_controller.rb]: `before_filter` will be deprecated in Rails5, let's use `before_action` from now on.

File: app/controllers/carto/kuviz/visualizations_controller.rb [hunk 2]
```diff
@@ -1,10 +1,14 @@
 module Carto
   module Kuviz
     class VisualizationsController < ApplicationController
+      include Carto::FrameOptionsHelper
```
Comment (alrocar) [inline app/controllers/carto/kuviz/visualizations_controller.rb]: Didn't know of this module, we can refactor `oauth_provider_controller` then to use it as well. Would you mind to include it?

File: app/controllers/carto/kuviz/visualizations_controller.rb (hunk 1 above)
Comment (Jesus89) [reply app/controllers/carto/kuviz/visualizations_controller.rb]: OK

File: app/controllers/carto/kuviz/visualizations_controller.rb (hunk 2 above)
Comment (Jesus89) [reply app/controllers/carto/kuviz/visualizations_controller.rb]: Yes

**Comment by Jesus89** [comment]
I have added the FrameOptionsHelper in `oauth_provider_controller` too.

**Comment by Jesus89** [comment]
Backend tests have failed, but I have checked the log and it seems that it's not related to the changes in this PR `¯\_(ツ)_/¯`

```
UserMigration api keys import and exports keeps roles for oauth api keys with schemas grants and you can drop tables after migration
     Failure/Error: expect(result.count).to eq 1
```

**Comment by gonzaloriestra** [comment]
@Jesus89 right, the test is not related. It's failing in all the branchs right now...

**Comment by alrocar** [comment]
I'm going to take a look at the test

**Comment by alrocar** [comment]
Acceptance 🍏

**Comment by jaimecartodb** [comment]
Might be worth having a conversation with Chelsea once she’s back… at least to understand where the disconnect came from.

**Comment by alrocar** [comment]
@Jesus89 ready to merge

**Comment by Jesus89** [comment]
VAMOS!

**Pull request merged**